src/main/kotlin/net/dungeonhub/application/service/BirthdayService.kt

```
@@ -119,32 +122,27 @@ object BirthdayService : StartupListener {
+                    val birthdayList = calendar.componentList.all.mapNotNull { component ->
+                        Birthday.fromComponent(component)
+                    }
+
+                    if (birthdayList.isNotEmpty()) {
+                        birthdays = birthdayList
                     }
                 }
             }
         } catch (exception: IOException) {
             logger.error(null, exception)
         } catch (exception: NullPointerException) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)